sdk/containerregistry/Azure.Containers.ContainerRegistry/src/ContainerRegistryClient.cs

```diff
@@ -255,7 +255,9 @@ public virtual Response DeleteRepository(string repositoryName, CancellationToke
         /// <returns> A new <see cref="ContainerRepository"/> for the desired repository. </returns>
         /// <exception cref="ArgumentNullException"> Thrown when <paramref name="repositoryName"/> is null. </exception>
         /// <exception cref="ArgumentException"> Thrown when <paramref name="repositoryName"/> is empty. </exception>
+#pragma warning disable AZC0004 // DO provide both asynchronous and synchronous variants for all service methods.
```

Comment:
Curious: why was this allowed before?  Is this a new analyzer warning, or did the existing analyzer warning change?  It would be helpful to update the PR description with the list of changes that are going in this PR.

Reply:
The existing analyzer warning change. I updated https://github.com/Azure/azure-sdk-tools/pull/6496#issue-1800304335